src/navigation/ActivismStack.js

```diff
@@ -0,0 +1,18 @@
+import { createStackNavigator } from 'react-navigation';
+import ActivismScreen from '../screens/ActivismScreen';
+
+import styles from './styles';
+
+const ActivismStack = createStackNavigator(
```

Comment:
I've personally only used `react-router` and thought we'd use `react-router-native`, though I'm sure this is fine.

Reply:
I honestly couldn't tell the difference between most of the popular navigation packages. I spent a long time trying to use `react-native-navigation` before figuring out that it's not compatible with our setup. The `react-navigation` api was easy enough for me to learn, and it seems popular and well supported on npm/github